playbooks/templates/.env.j2

```diff
@@ -55,8 +55,9 @@ PORTAL_NAME={{ webportal_server_config.portal_name }}
 # Possible choices:
 # - 'a': Accounts
 # - 'j': Jaeger
+# - 'm': MongoDB (when 'a' (i.e. Accounts) are used, this is included automatically)
 # Example:
-# PORTAL_MODULES=aj
+# PORTAL_MODULES=maj
```

Comment:
I wouldn't use `maj` as an example since people shouldn't use that. either `aj` or `mj`. I think we could use both as examples. 
```suggestion
# PORTAL_MODULES=aj
# or
# PORTAL_MODULES=mj
```